features/functests/sctp/sctp.go

```diff
@@ -104,7 +154,7 @@ func loadMC() *mcfgv1.MachineConfig {
 
 func checkForSctpReady(client *kubernetes.Clientset) {
 	nodes, err := client.CoreV1().Nodes().List(metav1.ListOptions{
-		LabelSelector: "node-role.kubernetes.io/worker=",
+		LabelSelector: "node-role.kubernetes.io/worker-sctp=",
```

Comment:
We may end up with a worker-A / worker-B machine pools, so this label is likely to change.